src/test/resources/fixtures/hmh-ri/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "type":"AssessmentEvent",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "actor":{
+    "id":"https://https://www.hmhco.com/ed/users/userRefId",
+    "type":"Person",
+    "name" : "Casandra Rath"
+  },
+  "action":"Started",
+  "object":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId",
+    "type":"Assessment",
+    "name":"HMH-RI Quiz Name - Reading Inventory",
+    "dateToStartOn":"2016-11-14T05:00:00.000Z",
```

Comment:
As an assignable Digital Resource an Assessment has multiple dates available to it.
I'd recommend looking at the full set and seeing if using more of the full set may help address some of our use cases. 
e.g. dateToActivate may indicate when a proctor/invigilator opened the assessment to the candidates, remember than the dateCreated doesn't reflect the data of the assessment content, it's when the assignment was created.